modules/code-editor/src/views/full/components/NewFileModal.tsx

```diff
@@ -39,7 +39,7 @@ const NewFileModal: FC<Props> = props => {
     await props.openFile({
       name: finalName,
       location: finalName,
-      content: props.selectedType === 'action' ? baseAction : ' ',
+      content: ['action_legacy', 'action_http'].includes(props.selectedType) ? baseAction : ' ',
```

Comment:
`baseAction` is the inner function in the .js file:

```js
const baseAction = `
  /**
   * Small description of your action
   * @title The title displayed in the flow editor
   * @category Custom
   * @author Your_Name
   * @param {string} name - An example string variable
   * @param {any} value - Another Example value
   */
  const myAction = async (name, value) => {\n    \n    \n  }

  myAction(args.name, args.value)
`

There is no reference to the SDK here. Both legacy and http actions can reuse `baseAction` as is.
```

Reply:
If you don't `return myAction`, then the bot will call the action and will not wait until it's completed before continuing to the next action. If you set something like event.temp.something and expect it in the next node, it will fail

Reply:
Fixed in e05043459ccd2626b5793e9c357029439525bcaa